### PR TITLE
New windows image 1.3.4

### PR DESCRIPTION
**Pull request by matthew-rollitt**
V-Ray for 3ds-Max 2019 RT renamed to GPU and updated to use the Build 4.02.05. Blender-DR needed a fix to work with the older image versions and the new versions due to a python now coming installed in the images.

### CODE REVIEW

**Review by timotheeguerin — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -35,11 +32,10 @@ steps:
   displayName: 'Install prerequisites'
 - script: |
     pushd '$(projectRoot)'
-    python runner.py "Tests/TestConfiguration.json" "$(PS_BATCH_ACCOUNT_NAME)" "$(BatchAccountKey)" "$(PS_BATCH_ACCOUNT_URL)" "$(PS_BATCH_ACCOUNT_SUB)" "$(PS_STORAGE_ACCOUNT_NAME)" "$(StorageAccountKey)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_CLIENT_ID)" "$(ServicePrincipalCredentialsSecret)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_TENANT)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_RESOUCE)" 
+    python runner.py "Tests/TestConfiguration.json" "$(PS_BATCH_ACCOUNT_NAME)" "$(templateRunnerBatchAccountKey)" "$(PS_BATCH_ACCOUNT_URL)" "$(PS_BATCH_ACCOUNT_SUB)" "$(PS_STORAGE_ACCOUNT_NAME)" "$(StorageAccountKey)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_CLIENT_ID)" "$(ServicePrincipalCredentialsSecret)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_TENANT)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_RESOUCE)"
```
Comment (smith1511) [inline azure-pipelines.yml]: Remove runner key

File: ncj/3dsmax/scripts/3dsmax.ps1 [hunk 2]
```diff
@@ -70,10 +70,33 @@ function SetupDistributedRendering
...
+        ElseIf($renderer -eq "VRayAdv")
+        {
+            echo "Adv 2018"
+            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_Adv""`r`n"
+            $script:pre_render_script_content += "if index == 1 then (r.system_distributedRender = true;r.system_vrayLog_level = 4; r.system_vrayLog_file = ""$vrayLogFile"")`r`n"
+
+        }
+    }
+    ElseIf($maxVersion -eq "2019"){
+        IF($renderer -eq "VRayRT"){
```
Comment (smith1511) [inline ncj/3dsmax/scripts/3dsmax.ps1:92]: Don't we need to support VRay Adv for 2019?

With 2019 do we support VRay 3.6 too?  Either way we'll need to support Adv.

File: ncj/3dsmax/scripts/3dsmax.ps1 [hunk 3]
```diff
@@ -70,10 +70,33 @@ function SetupDistributedRendering
...
+            echo "RT 2019"
```
Comment (smith1511) [inline ncj/3dsmax/scripts/3dsmax.ps1]: remove

File: ncj/3dsmax/scripts/3dsmax.ps1 [hunk 4]
```diff
@@ -70,10 +70,33 @@ function SetupDistributedRendering
...
+
     $script:pre_render_script_content += "if index != 1 then (print ""VRay renderer not used, please save the scene with a VRay renderer selected."")`r`n"
-    $script:pre_render_script_content += "index = findString rendererName ""V_Ray_RT_""`r`n"
-    $script:pre_render_script_content += "if index == 1 then (r.distributed_rendering = true) else (r.system_distributedRender = true;r.system_vrayLog_level = 4; r.system_vrayLog_file = ""$vrayLogFile"")`r`n"
+
+    If($maxVersion -eq "2018")
+    {
+        IF($renderer -eq "VRayRT")
+        {
+            echo "RT 2018"
```
Comment (smith1511) [inline ncj/3dsmax/scripts/3dsmax.ps1]: Remove

File: ncj/3dsmax/scripts/3dsmax.ps1 [hunk 5]
```diff
@@ -70,10 +70,33 @@ function SetupDistributedRendering
...
+            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_Adv""`r`n"
+            $script:pre_render_script_content += "if index == 1 then (r.system_distributedRender = true;r.system_vrayLog_level = 4; r.system_vrayLog_file = ""$vrayLogFile"")`r`n"
+
+        }
+    }
+    ElseIf($maxVersion -eq "2019"){
+        IF($renderer -eq "VRayRT"){
+            echo "RT 2019"
+            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_GPU_""`r`n"
+            $script:pre_render_script_content += "if index == 1 then (r.distributed_rendering = true) else (r.system_distributedRender = true;r.system_vrayLog_level = 4; r.system_vrayLog_file = ""$vrayLogFile"")`r`n"
```
Comment (smith1511) [inline ncj/3dsmax/scripts/3dsmax.ps1]: No point in having the else in here.

**Review by smith1511 — CHANGES_REQUESTED** [review]

**Pull request merged**